upload_form.py

```diff
@@ -459,8 +550,11 @@ def set_columns(filename, fileformat=None):
                                                     branch=branch_database,
                                                    )
     except Exception as ex:
-        return render_template('error.html', error=str(ex),
-                               traceback=traceback.format_exc(ex))
+        if testmode:
+            raise ex
+        else:
+            return render_template('error.html', error=str(ex),
```

Comment:
@keflavich - Maybe tack on a ```, None``` for when an error occurs. On line 495, a ValueError is raised (ie. Too many values to unpack) instead of linking to the error page.